### PR TITLE
imlib/draw: Fix out of bound crash issue with set_pixel_aa.

### PR DESCRIPTION
**Pull request by kwagyeman**
The previous code did not bounds check before reading. This can cause the system to crash.

### CODE REVIEW

File: src/omv/imlib/draw.c [hunk 1]
```diff
@@ -89,42 +89,46 @@ static void point_fill(image_t *img, int cx, int cy, int r0, int r1, int c) {
 }
 
 static void imlib_set_pixel_aa(image_t *img, int x, int y, int err, int c) {
-    // float alpha = 1 - err / 256.0;
```
Comment (iabdalkader) [inline src/omv/imlib/draw.c:92]: Can we instead check and return ?

```C
if (!((0 <= x) && (x < img->w) && (0 <= y) && (y < img->h))) {
    // Out of bounds.
    return;
}

Comment (kwagyeman) [reply src/omv/imlib/draw.c:92]: Sure, yeah. That's faster.

Comment (kwagyeman) [reply src/omv/imlib/draw.c:92]: Updated.

Comment (kwagyeman) [reply src/omv/imlib/draw.c:92]: Note, I removed imlib_set_pixel() because that would bounds check again.

**Review by iabdalkader — APPROVED** [review]

**Pull request merged**